apps/website/src/pages/docs/carousel.mdx

```diff
@@ -14,64 +14,80 @@ import * as AllExamples from 'examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='carousel' />
-
 <LiveExample src='Carousel.main.tsx'>
   <AllExamples.CarouselMainExample client:load />
 </LiveExample>
 
-The Carousel component consists of a set of slides, normally displayed one at a time. A navigation section is
-located below the slides, consisting of "dots" and "previous"/"next" buttons, used for changing slides.
+The Carousel component consists of a set of slides, normally displayed one at a time. A navigation section is located below the slides, consisting of ["dots"](#dots-list) and ["previous"/"next"](#navigation) buttons, used for changing slides.
+
+The Carousel component is made up of [customizable subcomponents](#component-api). Here's what the basic usage looks like:
```

Comment:
This paragraph and code snippet need to go under "Usage" heading

Reply:
no, i'm only talking about the second paragraph, as suggested in https://github.com/iTwin/iTwinUI/pull/1675#discussion_r1399344230. the first paragraph is unrelated to "Usage"